core/domain/stats_jobs_one_off.py

```diff
@@ -214,6 +214,18 @@ def entity_classes_to_map_over(cls):
 
     @classmethod
     def prepare_map(cls, item):
+        """Returns a dict that represents the given model instance, so that it
+        can be processed by the MapReduce pipeline.
+
+        Args:
+            item: *. The model instance for different statistics models.
```

Comment:
Enumerate the possible types in the typeinfo.

Reply:
Done